BedrockServer.cpp

```diff
@@ -597,6 +592,16 @@ void BedrockServer::worker(SData& args,
                                 core.rollback();
                             } else {
                                 bool commitSuccess;
```

Comment:
Can you move this beneath your comment block, right next to the declaration of `shared_lock`?  That'll keep it all together.